pkg/deploy/controller/imagechange/controller.go

```diff
@@ -111,10 +111,10 @@ func (c *ImageChangeController) Handle(stream *imageapi.ImageStream) error {
 	}
 
 	if anyFailed {
-		return fatalError(fmt.Sprintf("couldn't update some deployment configs for trigger on image stream %q", imageapi.LabelForStream(stream)))
```

Comment:
I have been meaning to fix this but the image change controller will not be a thing once https://github.com/openshift/origin/pull/9349 merges.